apps/dcos_overlay/src/dcos_overlay_poller.erl

```diff
@@ -339,6 +493,34 @@ parse_vtep_mac(MAC) ->
         end,
         MACComponents).
 
+%%%===================================================================
+%%% Metrics functions
+%%%===================================================================
+
+-spec(init_metrics() -> ok).
+init_metrics() ->
+    prometheus_counter:declare([
+        {registry, overlay},
+        {name, mesos_state_polls_total},
```

Comment:
I think it's a misleading name, we don't poll mesos state here. I'd suggest `polls_total` or `local_polls_total` or a summary metric (e.g `poll_duration_seconds`; like there is a `call_duration_seconds` metric in `dcos_net_mesos` module for operatior api calls).